src/main/java/com/gmail/nossr50/util/commands/CommandRegistrationManager.java

```diff
@@ -400,6 +401,15 @@ private static void registerMcImportCommand() {
         command.setExecutor(new McImportCommand());
     }
 
+    private static void registerReloadLocaleCommand() {
+        PluginCommand command = mcMMO.p.getCommand("mcmmoreloadlocale");
+        command.setDescription("Reloads locale"); // TODO: Localize
```

Comment:
o rly

Reply:
I just followed the style, something needs to be left for you to do as well :^)